stock_request/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Stock Request",
     "summary": "Internal request for stock",
-    "version": "11.0.3.1.0",
+    "version": "11.0.3.2.0",
```

Comment:
@BT-rmartin You could also change the patch part (as the modification is not so great)